### PR TITLE
I18n for flag

### PR DESCRIPTION
**Pull request by hin101**
Added i18n translations for flag feature.

### CODE REVIEW

**Comment by hin101** [comment]
Reference to issue #447

File: app/views/posts/_post.html.erb [hunk 1]
```diff
@@ -71,14 +71,14 @@
     <div class="modal-content">
       <div class="modal-header">
         <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-        <h4 class="modal-title">Flag Post for Review</h4>
+        <h4 class="modal-title"><%= t(:flag_post_for_review) %></h4>
```
Comment (scott) [inline app/views/posts/_post.html.erb]: We generally provide a `default` for each of the `i18n` strings that is equal to what is provided in the `en.yml` file.

File: app/views/posts/_post.html.erb [hunk 2]
```diff
@@ -71,14 +71,14 @@
...
       </div>
 
       <div class="modal-body row">
         <div class="col-md-12">
         <%= bootstrap_form_for :flag, :url => post_flags_path(post) do |f| -%>
-          <%= f.text_area :reason, rows: 6, cols: 160, placeholder: "Describe the reason for flagging this post", skip_label: true -%>
-          <%= f.submit 'Flag', class: 'btn btn-warning' -%>
+          <%= f.text_area :reason, rows: 6, cols: 160, placeholder: t(:flag_post_placeholder), skip_label: true -%>
+          <%= f.submit t(:flag), class: 'btn btn-warning' -%>
```
Comment (scott) [inline app/views/posts/_post.html.erb]: Same here, can you add `default:`

File: config/locales/en.yml [hunk 3]
```diff
@@ -82,6 +82,12 @@ en:
   get_your_copy: "Grab your copy on GitHub"
   version: "Version:"
 
+  # flag
+  flag_for_review: "Flag for Review"
+  flag_post_for_review: "Flag Post for Review"
+  flag_post_placeholder: "Describe the reason for flagging this post"
+  flag: "Flag"
+
```
Comment (scott) [inline config/locales/en.yml]: Nice work! Can you also add these keys to the other `yml` files and just comment them out to indicate that they need to be translated?

**Pull request merged**